src/AudacityApp.cpp

```diff
@@ -1460,6 +1461,8 @@ bool AudacityApp::InitPart2()
       SplashDialog::DoHelpWelcome(*project);
    }
 
+   mUpdateManager = std::make_unique<UpdateManager>(*project);
```

Comment:
This is where you should find the project's frame, and just pass that instead.

This is happening only when the application initializes, and the project is the empty untitled project that is always made then.

Study of UpdateManager tells me it really only wants a window.  That should be more obvious just from reading this make_unique call.